bookofmudora.asm

```diff
@@ -28,90 +28,89 @@ RTL
 ; Randomize Bonk Keys
 ;--------------------------------------------------------------------------------
 LoadBonkItemGFX:
-	LDA.b #$08 : STA.w SpriteOAMProp, X ; thing we wrote over
+    LDA.b #$08 : STA.w SpriteOAMProp, X ; thing we wrote over
 LoadBonkItemGFX_inner:
-	LDA.b #$00 : STA.l RedrawFlag
-	JSR LoadBonkItem_Player : STA !MULTIWORLD_SPRITEITEM_PLAYER_ID
-	JSR LoadBonkItem
-        JSL.l AttemptItemSubstitution
-        JSL.l ResolveLootIDLong
-        STA.w SpriteItemType, X
-        STA.w SpriteID, X
-	JSL.l PrepDynamicTile
-        PHA : PHX
-        LDA.w SpriteID,X : TAX
-        LDA.l SpriteProperties_standing_width,X : BNE +
-                LDA.b #$00 : STA.l SpriteOAM : STA.l SpriteOAM+8
-        +
-        PLX : PLA
+    LDA.b #$00 : STA.l RedrawFlag
+    JSR LoadBonkItem_Player : STA.l !MULTIWORLD_SPRITEITEM_PLAYER_ID
+    JSR LoadBonkItem
+    JSL AttemptItemSubstitution
+    JSL ResolveLootIDLong
+    STA.w $0E80, X
```

Comment:
Why remove this label?

Reply:
This is something I changed in my GFX system changes. This is one of those sprite variables that are used differently for other sprite types, but in rando we hijack it for something else. So, I was planning on waiting to address this in a later PR instead of putting in effort towards something that's changing later anyways. I can go back and modify this if you want, but I figured there was no point.

Reply:
Good enough for me.